### DISCUSSION TITLE
Unable to style DataTable component using pt

### DISCUSSION THREAD

**Discussion by Uzeyir9695** [primefaces/community#1462] [PrimeVue] [answered]
Could you please show me the way to style `DataTable` component's `thead and paginator` using `pt`.
Please visit this link, I added some tailwind class to the `thead` but it didn't affect. Also show me the example how to style `paginator` of its component not an actual `Paginator` component.

`<DataTable
      pt:thead:class="text-3xl font-black bg-slate-300"
      :value="customers"
      paginator
      :rows="5"
      :rowsPerPageOptions="[5, 10, 20, 50]"
      tableStyle="min-width: 50rem"
    >
      <Column field="name" header="Name" style="width: 25%"></Column>
      <Column field="country.name" header="Country" style="width: 25%"></Column>
      <Column field="company" header="Company" style="width: 25%"></Column>
      <Column
        field="representative.name"
        header="Representative"
        style="width: 25%"
      ></Column>
    </DataTable>`

[Stackblitz](https://stackblitz.com/edit/dntvm6?file=src%2FApp.vue)

**Comment by tugcekucukoglu** [comment, marked as answer]
```
:pt="{
    thead: 'text-3xl',
   paginator: {
       root: 'bg-red-300'
  }
 }"
```
